api/iterator.go

```diff
@@ -0,0 +1,60 @@
+package api
+
+import (
+	"fmt"
+	dbm "github.com/tendermint/tm-db"
+)
+
+// This stores all Iterators for one contract
+type Iterators []dbm.Iterator
+
+// IteratorStack contains one entry for each contract, LIFO style
+var IteratorStack []Iterators
```

Comment:
If these declarations are not used outside this module, i think they should be made private.
Go's naming scheme will make these look confusing (to me, at least) but i think it's the right thing to do.

Reply:
renamed and made private